tutorials/on-beyond-magpie0.md

```diff
@@ -0,0 +1,111 @@
+---
+title: On Beyond Magpie 0: Setup and API exploration
+description: An introduction to the Cloud Natural Language API, aimed at Advanced Placement Computer
+Science classes who have worked on the Magpie lab, but suitable for most people starting with the
+Cloud Natural Language API.  Demonstrates how to access the Cloud Natural Language API interatively and create 
+credentials for use with later tutorials in this sequence.
+author: lauriewhite
+tags: Cloud Natural Language API, APCS, REST, Magpie, education
+date_published: 2017-03-28
+---
+# On Beyond Magpie, Part 0: Setup and API exploration
+
+The Advanced Placement Computer Science A program provides 
+the [Magpie lab](http://media.collegeboard.com/digitalServices/pdf/ap/ap-compscia-magpie-lab-student-guide.pdf) for students to practice using basic control structures to parse user input as part of a chatbot.  This tutorial is designed to be an additional enrichment exercise (typically used after the AP exam) to go beyond  basic parsing and instead use [Google's Cloud Natural Language API][nlp], a pretrained machine learning model that will do text analysis for the user.  The lab demonstrates how to use the Cloud Natural Language API to extend the Magpie lab.
```

Comment:
I'm not sure if it's proper etiquette to reply to each of these comments individually.  If so, I screwed up and just responded to those where the response was anything other than "Thanks, fixed it."